repo/packages/C/concord/0/marathon.json.mustache

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
you probably also want a label that specifies the dcos package name, ala https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/C/cassandra/1/marathon.json.mustache#L35